src/Android/Accessibility/AccessibilityService.cs

```diff
@@ -54,106 +57,95 @@ public override void OnAccessibilityEvent(AccessibilityEvent e)
 
                 if(SkipPackage(e?.PackageName))
                 {
-                    CancelOverlayPrompt();
-                    return;
-                }
-
-                var root = RootInActiveWindow;
-                if(root == null || root.PackageName != e.PackageName)
-                {
+                    if(e?.PackageName != "com.android.systemui")
+                    {
+                        CancelOverlayPrompt();
+                    }
                     return;
                 }
 
                 // AccessibilityHelpers.PrintTestData(root, e);
 
+                AccessibilityNodeInfo root = null;
+
                 switch(e.EventType)
                 {
                     case EventTypes.ViewFocused:
                     case EventTypes.ViewClicked:
-                    case EventTypes.ViewScrolled:
+                        if(e.Source == null || e.PackageName == BitwardenPackage)
+                        {
+                            CancelOverlayPrompt();
+                            e.Recycle();
+                            break;
+                        }
+
+                        root = RootInActiveWindow;
+                        if(root == null)
```

Comment:
I would just combine this with the check below, like it was originally.

Also, do we need this kind of check in the other switch case below? I don't see us doing this check there.

```
if(root == null || root.PackageName != e.PackageName)
                        {
                            e.Recycle();
                            break;
                        }

```